doc/develop/fixing_data.rst

```diff
@@ -326,30 +333,66 @@ strictness to the highest:
 Natively supported non-CMIP datasets
 ====================================
 
-Fixed datasets are supported through the ``native6`` project.
-Put the files containing the data in the directory that you have configured
-for the ``native6`` project in your :ref:`user configuration file`, in a
-subdirectory called ``Tier{tier}/{dataset}/{version}/{frequency}/{short_name}``.
-Replace the items in curly braces by the values used in the variable/dataset
-definition in the :ref:`recipe <recipe_overview>`.
-Below is a list of datasets currently supported.
+Some fixed datasets and native models formats are supported through
+the ``native6`` project or through a dedicated project.
 
-ERA5
-----
+.. _fixing_native_models:
 
-- Supported variables: ``clt``, ``evspsbl``, ``evspsblpot``, ``mrro``, ``pr``, ``prsn``, ``ps``, ``psl``, ``ptype``, ``rls``, ``rlds``, ``rsds``, ``rsdt``, ``rss``, ``uas``, ``vas``, ``tas``, ``tasmax``, ``tasmin``, ``tdps``, ``ts``, ``tsn`` (``E1hr``/``Amon``), ``orog`` (``fx``)
-- Tier: 3
+Native models : IPSL-CM6,... 
+-----------------------------
 
-MSWEP
------
+The following models are natively supported through the procedure
+described above (:ref:`fix_structure`) and at
+:ref:`configure_native_models`:
 
-- Supported variables: ``pr``
-- Supported frequencies: ``mon``, ``day``, ``3hr``.
-- Tier: 3
+  - **IPSL-CM6** : both output formats (i.e. the ``Output`` and the
+    ``Analyse / Time series`` formats) are supported, and should be
+    configured in recipes as e.g.:
 
-For example for monthly data, place the files in the ``/Tier3/MSWEP/latestversion/mon/pr`` subdirectory of your ``native6`` project location.
+    .. code-block:: yaml
 
-.. note::
-  For monthly data (V220), the data must be postfixed with the date, i.e. rename ``global_monthly_050deg.nc`` to ``global_monthly_050deg_197901-201710.nc``
+      datasets:
+        - {simulation: CM61-LR-hist-03.1950, exp: piControl, freq: Analyse/TS_MO,
+           account: p86caub,  status: PROD, dataset: IPSL-CM6, project: native6,
+	   root: /thredds/tgcc/store} 
```

Comment:
Is `root` describing some input path? Is that still needed with the addition of the extra facets file?

Reply:
Yes, that is still needed because , for model development use case, modelers have some choice on where to put the data, which may change from one experiment to the other